### PR TITLE
[ISSUE #22] improving codeCov of mqtt.ds

### LINKED ISSUE #22: codeCov is not very optimistic, need to increase code coverage
in branch main codeCov 20%
 need add code coverage
image

### PR DESCRIPTION
**Pull request by YxAc**
continuously fix #22

### CODE REVIEW

File: mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/FirstTopicManager.java [hunk 1]
```diff
@@ -82,6 +82,11 @@ public void init() throws MQClientException {
         }, 0, 5, TimeUnit.SECONDS);
     }
 
+    public void initMQAdminExt() throws MQClientException {
```
Comment (YxAc) [inline mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/FirstTopicManager.java:85]: take it out for ut

File: mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/FirstTopicManager.java [hunk 2]
```diff
@@ -108,6 +113,9 @@ public void checkFirstTopicIfCreated(String firstTopic) {
     }
 
     private void updateTopicRoute(String firstTopic) {
+        if (StringUtils.isBlank(firstTopic)) {
```
Comment (YxAc) [inline mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/FirstTopicManager.java:116]: bug of blank judgment

File: mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/mq/MqAdmin.java [hunk 3]
```diff
@@ -28,9 +28,13 @@ public class MqAdmin {
     private DefaultMQAdminExt defaultMQAdminExt;
 
     public MqAdmin(Properties properties) {
+        this(properties.getProperty("NAMESRV_ADDR"));
```
Comment (YxAc) [inline mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/mq/MqAdmin.java:31]: add this constructor for optimizing MqFactory to reuse the same code block

File: mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/mq/MqFactory.java [hunk 4]
```diff
@@ -58,55 +55,28 @@ public static synchronized DefaultMQPullConsumer buildDefaultMQPullConsumer(Stri
...
-        defaultMQPushConsumer.setInstanceName(buildInstanceName());
-        defaultMQPushConsumer.setVipChannelEnabled(false);
-        defaultMQPushConsumer.setConsumerGroup(group);
-        if (messageListener instanceof MessageListenerOrderly) {
-            defaultMQPushConsumer.registerMessageListener((MessageListenerOrderly) messageListener);
-        } else {
-            defaultMQPushConsumer.registerMessageListener((MessageListenerConcurrently) messageListener);
-        }
-        return defaultMQPushConsumer;
+        MqConsumer mqConsumer = new MqConsumer(properties, nameSrv);
```
Comment (YxAc) [inline mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/mq/MqFactory.java:65]: reusing the same code block

File: mqtt-ds/src/test/java/org/apache/rocketmq/mqtt/ds/test/store/TestLmqOffsetStoreManager.java [hunk 5]
```diff
@@ -51,12 +51,8 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-/**
- * @author tianliuliu
- * @date 2022-03-14 11:50
- */
 @RunWith(MockitoJUnitRunner.class)
-public class TestOffsetStoreManager {
+public class TestLmqOffsetStoreManager {
```
Comment (YxAc) [inline mqtt-ds/src/test/java/org/apache/rocketmq/mqtt/ds/test/store/TestLmqOffsetStoreManager.java:55]: renaming

**Review by tianliuliu — APPROVED** [review]

**Pull request merged**